src/main/scala/com/databricks/spark/csv/package.scala

```diff
@@ -23,19 +23,22 @@ package object csv {
    * Adds a method, `csvFile`, to SQLContext that allows reading CSV data.
    */
   implicit class CsvContext(sqlContext: SQLContext) {
-    def csvFile(filePath: String) = {
+    def csvFile(filePath: String,
```

Comment:
Common case is handled with default parameters. Why should we limit the options to users?